Tests/FunctionalTests/PluginTests.swift

```diff
@@ -925,15 +956,15 @@ final class PluginTests: XCTestCase {
                         do {
                             try scriptRunner.cancel(deadline: .now() + .seconds(5))
                         } catch {
-                            XCTFail("Cancelling script runner should not fail: \(error)")
+                            fatalError("Cancelling script runner should not fail: \(error)")
                         }
                     }
                 }
                 group.addTask {
                     do {
                         try await Task.sleep(nanoseconds: UInt64(DispatchTimeInterval.seconds(3).nanoseconds()!))
                     } catch {
-                        XCTFail("The plugin should not finish within 3 seconds")
+                        try #require(true == false, "The plugin should not finish within 3 seconds")
```

Comment:
issue (blocking)" replace the `try #require(...)` with `Issue.record("The plugin should not finish within 3 seconds")`.

See https://developer.apple.com/documentation/testing/migratingfromxctest